Mathlib/Probability/Distributions/Gaussian/Basic.lean

```diff
@@ -170,6 +170,28 @@ alias ⟨_, isGaussian_of_charFunDual_eq⟩ := isGaussian_iff_charFunDual_eq
 
 end charFunDual
 
+section charFun
+
+open InnerProductSpace
+open scoped RealInnerProductSpace
+
+variable {E : Type*} [NormedAddCommGroup E] [InnerProductSpace ℝ E] [MeasurableSpace E]
+    [CompleteSpace E] [BorelSpace E] {μ : Measure E}
+
+lemma isGaussian_iff_charFun_eq [IsFiniteMeasure μ] :
+    IsGaussian μ ↔
+    ∀ t, charFun μ t = exp (μ[fun x ↦ ⟪t, x⟫] * I - Var[fun x ↦ ⟪t, x⟫; μ] / 2) := by
```

Comment:
In fact I think this result should also be true without completeness by continuity of `charFunDual` in the variable `L`. But we don't know that `toDualMap` has dense range in general, so could you just leave a TODO?